app/views/layouts/angular/_auth_credentials_angular.html.haml

```diff
@@ -1,7 +1,7 @@
 - validate_url     ||= 'log_depot_edit'
 - prefix           ||= 'log'
-- uid_label        ||= "Username"
-- pwd_label        ||= "Password"
+- uid_label        ||= _("Username")
```

Comment:
If you have to make another change to this PR, could also rename `uid_label` to `username_label` or something?  I see `uid` and I think "unique_identifier", not "user id"